recipes/certifi/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2016.2.28" %}
+
+package:
+  name: certifi
+  version: {{ version }}
+
+source:
+  fn: certifi-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/c/certifi/certifi-{{ version }}.tar.gz
+  sha256: 1
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - certifi
+
+about:
+  home: http://certifi.io/
+  license: ISC
+  summary: Python package for providing Mozilla's CA Bundle.
+
+extra:
+  recipe-maintainers:
+    - dan-blanchard
```

Comment:
It used to, but now it just warns:

https://travis-ci.org/conda-forge/conda-forge.github.io/jobs/122567637:

```
Running command: ['python', '/home/travis/build/conda-forge/conda-forge.github.io/scripts/update_teams.py', './feedstocks_repo/feedstocks']
AN OLD MEMBER (dan-blanchard) NEEDS TO BE REMOVED FROM conda-forge/certifi-feedstock
THE conda-forge-webservices TEAM NEEDS TO BE REMOVED.
Keeping  Core
Keeping  all-members
Keeping  conda-forge.github.io
```

The message about the conda-forge-webservices team also needs updating.